Traceback (most recent call last):
  File "f:\jinzhu\conda_env\luto\Lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "f:\jinzhu\conda_env\luto\Lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "f:\jinzhu\conda_env\luto\Lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "f:\jinzhu\conda_env\luto\Lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "f:\jinzhu\conda_env\luto\Lib\asyncio\base_events.py", line 687, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "f:\jinzhu\conda_env\luto\Lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "f:\jinzhu\conda_env\luto\Lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "f:\jinzhu\conda_env\luto\Lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# REMOVE
yr_idx = year_target - year_begin
# Get the quantity matrices
q_mrp = get_quantity_matrices(data, yr_idx).astype(np.float32)          # t/cell
q_mrp_xr = mrj_to_xr(data, q_mrp).expand_dims({'year': [year_target]})  
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mNameError[0m                                 Traceback (most recent call last)
Cell [1;32mIn[1], line 2[0m
[0;32m      1[0m [38;5;66;03m# REMOVE[39;00m
[1;32m----> 2[0m yr_idx [38;5;241m=[39m [43myear_target[49m [38;5;241m-[39m year_begin
[0;32m      3[0m [38;5;66;03m# Get the quantity matrices[39;00m
[0;32m      4[0m q_mrp [38;5;241m=[39m get_quantity_matrices(data, yr_idx)[38;5;241m.[39mastype(np[38;5;241m.[39mfloat32)          [38;5;66;03m# t/cell[39;00m

[1;31mNameError[0m: name 'year_target' is not defined
NameError: name 'year_target' is not defined

